### PR TITLE
Deprecate julia in favor of juliaup

### PR DESCRIPTION
**Pull request by GZGavinZhao**
## Reason
_Reason for deprecation or undeprecation_

getsolus/packages#266

## Does this request depend on package changes to land first?
_This request depends on a package change to land before this can be merged._

[x] Yes

## Package Diff
_The URL to the package change this depends on, if any_

getsolus/packages#363

### CODE REVIEW

File: repo_data/distribution.xml.in [hunk 1]
```diff
@@ -2778,5 +2778,8 @@
 
 		<!-- Replaced by eza -->
 		<Package>exa</Package>
+
+		<!-- Replaced by juliaup -->
+		<Package>julia</Package>
```
Comment (ReillyBrogan) [inline repo_data/distribution.xml.in:2783]: You need to do _all_ of the packages built by the julia repo. From looking at the [unstable repo](https://packages.getsol.us/unstable/j/julia/) I see `julia`, `julia-devel`, `julia-docs`, and `julia-dbginfo`.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: repo_data/distribution.xml.in (hunk 1 above)
Comment (GZGavinZhao) [reply repo_data/distribution.xml.in:2783]: Aw shoot made the same mistake I did with Haskell.

**Review by ReillyBrogan — APPROVED** [review]
LGTM, holding until linked PR is merged/built and it's time for the sync.

**Review by ReillyBrogan — APPROVED** [review]
LGTM, thanks!

**Pull request merged**